Automattic/jetpack
# Use PlanIcon component in the My Plan section

### Changes proposed in this Pull Request

This PR gets rid of the references to products and plans icons in the _My Plan_ section, in favor of the _PlanIcon_ component.

Fixes 1164141197617539-as-1151269793214129

### Implementation notes

A next step could be to import `ProductIcon` from `@automattic/components`. For the moment, icons have been copied from Calypso to this PR.

### Testing instructions

- Run the plugin locally
- Visit `/wp-admin/admin.php?page=jetpack#/my-plan`
- Choose a couple of plans/products, and verify that you see its icon (see capture below)

### Screenshots
![merge_from_ofoct](https://user-images.githubusercontent.com/1620183/94027618-08e82580-fd89-11ea-9f13-8b472d4a44a0.jpg)



### Does this pull request change what data or activity we track or use?
No

### Proposed changelog entry for your changes:
Refactor icons in the My Plan section
